### ISSUE TITLE
throw new assert.AssertionError({  while linting

### ISSUE THREAD

**Issue by whomba** [sasstools/gulp-sass-lint#28]
# Issue

I am receiving the following errors:

```
catFace:~/tests/gulp$ gulp lintSCSS
[14:38:19] Using gulpfile ~/tests/gulp/gulpfile.js
[14:38:19] Starting 'lintSCSS'...
[14:38:19] Finished 'lintSCSS' after 8.31 ms

assert.js:89
  throw new assert.AssertionError({
  ^
AssertionError: missing path
    at Module.require (module.js:351:3)
    at require (internal/module.js:12:17)
    at module.exports (/home/local/catFace/tests/gulp/node_modules/sass-lint/lib/config.js:73:12)
    at Function.sassLint.getConfig (/home/local/catFace/tests/gulp/node_modules/sass-lint/index.js:25:10)
    at DestroyableTransform._transform (/home/local/catFace/tests/gulp/node_modules/gulp-sass-lint/index.js:34:19)
    at DestroyableTransform.Transform._read (/home/local/catFace/tests/gulp/node_modules/through2/node_modules/readable-stream/lib/_stream_transform.js:172:10)
    at DestroyableTransform.Transform._write (/home/local/catFace/tests/gulp/node_modules/through2/node_modules/readable-stream/lib/_stream_transform.js:160:12)
    at doWrite (/home/local/catFace/tests/gulp/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:335:12)
    at writeOrBuffer (/home/local/catFace/tests/gulp/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:321:5)
    at DestroyableTransform.Writable.write (/home/local/catFace/tests/gulp/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:248:11)
```
# Expected

My amazing foo.scss style is properly linted
# Versions
- gulp CLI - 1.2.1
- gulp - 3.9.1
- sass-lint - 1.5.0
- gulp-sass-lint - 1.1.1 
# Files:
## gulpfile.js

```
// Dependencies
var gulp = require('gulp');
var sass = require('gulp-sass');
var sassLint = require('gulp-sass-lint');

gulp.task('lintSCSS', function(){
  gulp.src('src/assets/stylesheets/*.scss')
   .pipe(sassLint({config:".sass-lint.yml"})); 
});
```
## .sass-lint.yml

```
files:
  include: 'src/assets/stylesheets/**/*.scss'
options:
  formatter: stylish
  merge-default-rules: false
rules:
  border-zero: 1
  trailing-semicolon: 1
```
## foo.scss

```
$foo: #f00;
body{
border:none
diplay:none}
```
## Directory tree:

Contents of node_modules is omitted, there's a bunch in there...

```
catFace:~/tests/gulp$ ls -a
.  ..  .sass-lint.yml  .scss-lint.yml  gulpfile.js  node_modules  src

catFace:~/tests/gulp$ tree
.
|-- gulpfile.js
`-- node_modules
`-- src
    `-- assets
        `-- stylesheets
            `-- foo.scss
```

**Comment by whomba** [comment]
::Ping::

**Comment by ngdio** [comment]
same problem here

**Comment by franmartosr** [comment]
I have the same problem too

**Comment by ghost** [comment]
There's no 'config' option for gulp-sass-lint, sass-lint use 'config-file' so it should be 'configFile' in Javascript notation. Have you tried to remove the config option?

**Comment by DanPurdy** [comment]
This was due to an error in sass-lint where if a config file couldn't be found it checked your package.json for the location of one. If your package file didn't exist or didn't include the location of a config file it failed.. Silly error but fixed since `sass-lint` 1.6.0

You can see the issue on sass-lint here https://github.com/sasstools/sass-lint/issues/563

Will close for now but if you see this re occuring let us know.
